### ISSUE TITLE
bed files

### ISSUE THREAD

**Issue by bjarnivh** [mccoy-lab/t2t-variants#1]
The bed files used here:
https://github.com/mccoy-lab/t2t-variants/blob/main/distance_to_novel-nonsyntenic/intersect_novel_and_nonsyntenic.sh
are not distributed with the project.  Is it possible to access them somewhere?

**Comment by dtaylo95** [comment]
Hello! Thanks for the question. The previously unresolved bed file is accessible through the UCSC assembly hub browser for the T2T project and is described here: [https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=1233948703_ZFOcFP7TeC49XE5ajuTQc7BgMgal&g=hub_2395475_chm13_uncovered_byGRCh38](https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=1233948703_ZFOcFP7TeC49XE5ajuTQc7BgMgal&g=hub_2395475_chm13_uncovered_byGRCh38). The bed file describing the regions non-syntenic with the GRCh38 assembly will be available on the UCSC assembly hub browser soon, but in the mean time, we've added this file in repo in the `distance_to_novel-nonsyntenic` directory.

We've also updated the README in that directory with the information here, as well as a direct link to download the bed file containing previously-unresolved regions.
